### ISSUE TITLE
Annotation on bar chart doesn't work properly.

### ISSUE THREAD

**Issue by HenryYan2012** [blacklabel/annotations#54]
1. Create a annotation on a horizontal bar chart.
![before resize browser](https://cloud.githubusercontent.com/assets/3112947/14550340/633b2b7e-02fa-11e6-9f8b-fc2e13eee79f.jpg)
2. View it on a smaller browser. You will see the position is in correct.
![after rezie browser](https://cloud.githubusercontent.com/assets/3112947/14550348/7f6eb3f6-02fa-11e6-80fb-5afbc7ec050e.jpg)

**Comment by pawelfus** [comment]
Could you setup jsfiddle with the issue? Have you set `xValue` and `yValue` to make sure annotation is linked to x-y vlaue, not x-y (pixels) position? See [API](http://www.highcharts.com/plugin-registry/single/17/Annotations).

**Comment by HenryYan2012** [comment]
@pawelfus Please check http://jsfiddle.net/z4eg8pmr/ You could add an circle annotation on it, and resize the result window. Then will see the issue.
I believe it is linked to x-y value.

**Comment by pawelfus** [comment]
Indeed, looks buggy. When using column series (not inverted chart) it works fine. 

Thank you for reporting!

**Comment by pawelfus** [comment]
*Internal note:*

Looks like values in drop event are not uodated and group still has `translate()` in the SVG. Other thing to check is we have proper axes when inverted chart.
